dev/com.ibm.ws.microprofile.metrics/resources/com/ibm/ws/microprofile/metrics/resources/Metrics.nlsprops

```diff
@@ -118,13 +118,24 @@ connectionpool.destroy.total.description=The total number of managed connections
 connectionpool.managedConnections.description=The number of managed connections in the free, shared, and unshared pools.
 connectionpool.connectionHandles.description=The number of connections that are in use. This number might include multiple connections that are shared from a single managed connection.
 connectionpool.freeConnections.description=The number of managed connections in the free pool.
+connectionpool.waitTime.total.description=The total wait time on all connection requests since the start of the server.
+connectionpool.inUseTime.total.description=The total time that all connections are in-use since the start of the server.
+connectionpool.usedConnections.total.description=The total number of used connections.
+connectionpool.queuedRequests.total=The total number of requests waited in the queue before a connection is granted.
```

Comment:
I think a couple of words could be updated so it's something like: The total number of requests that waited in the queue before a connection was granted.